src/System.Data.SqlClient/pkg/System.Data.SqlClient.pkgproj

```diff
@@ -3,13 +3,9 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Data.SqlClient.csproj">
-      <SupportedFramework>net461;netcoreapp2.0;$(UAPvNextTFM);$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>net461;netcoreapp2.0;uap10.0.16299;$(UAPvNextTFM);$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
@ericstj Do I need to add the older uap version here? If I leave it out, then I get a bunch of errors like the following (for the various win10 builds):
System.Data.SqlClient should support API version 4.2.0.0 on UAP,Version=v10.0.16299/win10-x86 but CoreFx\bin\ref\System.Data.SqlClient\4.4.0.0\netstandard\System.Data.SqlClient.dll was found to support 4.4.0.0.

Reply:
Yeah you need the older version.  That is declaring where you expect your reference to apply.  Since its a netstandard2 ref it has to apply to all frameworks that support ns2, including the older UAP.